Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM rustlang/rust:nightly-bookworm-slim
+
+RUN set -eux; \
+    apt-get update; \
+    apt-get install -y --no-install-recommends \
+        build-essential \
+        libclang-dev \
+        libssl-dev \
+        pkg-config
+
+COPY ./src ./src
```

Comment:
Is it possible to copy everything instead of file by file?

Reply:
You get easily tripped by `target` which is _huge_. Avoiding garbage in container also helps with disk space + build time.